packs/feats/crystal-luminescence.json

```
@@ -25,7 +25,40 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Impossible Lands"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "TokenLight",
+                "predicate": [
+                    "crystal-luminescence"
+                ],
+                "value": {
+                    "animation": {
+                        "type": "torch"
+                    },
+                    "bright": "20+@item.flags.pf2e.rulesSelections.crystalLuminescence",
+                    "color": "#1e0ab8",
+                    "dim": "40+@item.flags.pf2e.rulesSelections.crystalLuminescence"
+                }
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "crystal-luminescence",
+                "selection": "0",
+                "suboptions": [
+                    {
+                        "label": "PF2E.SpecificRule.CrystalLuminescence.Normal",
+                        "value": "0"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.CrystalLuminescence.Enhanced",
+                        "value": "10"
+                    }
+                ],
+                "toggleable": true,
+                "value": true
+            }
```

```suggestion
            {
                "domain": "all",
                "key": "RollOption",
                "option": "crystal-luminescence",
                "suboptions": [
                    {
                        "label": "PF2E.SpecificRule.CrystalLuminescence.Normal",
                        "value": "0"
                    },
                    {
                        "label": "PF2E.SpecificRule.CrystalLuminescence.Enhanced",
                        "value": "10"
                    }
                ],
                "toggleable": true
            }
```
Don't need preset values in these stored in compendium.